app/Models/Solo/Score.php

```diff
@@ -169,6 +169,13 @@ public function scopeDefault(Builder $query): Builder
         return $query->whereHas('beatmap.beatmapset');
     }
 
+    public function scopeForListing(Builder $query): Builder
+    {
+        return $query->where('ranked', true)
```

Comment:
is it worth/necessary forcing order to use `id` as a safety, as it's required for the [processing check](https://github.com/ppy/osu-web/pull/11705/commits/0539bead093cfa755251e3c30d969add1d710b45#diff-0eb62e470bf84ab233b39bd2f4a9abbd0e7fee8768a535f175336d480cd6db86R186-R190) to work?

Reply:
the order is applied somewhere else

Reply:
figured that was probably the case, cool.